setup.py

```diff
@@ -23,5 +23,6 @@
 setuptools.setup(
     packages=setuptools.find_packages(),
     pbr=True,
-    setup_requires=['pbr']
+    setup_requires=['pbr', 'setuptools>=12']
```

Comment:
I'm here to tell you that this doesn't make sense because of the way it works. `setuptools.setup()` installs this only for setup-time and then discards those.
But the problem is that it will not pick up a newer version because the older version is already in runtime and module cannot substitute itself.
The proper place for adding build deps nowadays is `pyproject.toml`->`[build-system]`->`requires'.